src/Catalyst.Abstractions/Kvm/Models/BlockParameters.cs

```diff
@@ -22,63 +22,98 @@
 #endregion
 
 using System;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
 using Nethermind.Core.Json;
 
 namespace Catalyst.Abstractions.Kvm.Models
 {
-    public sealed class BlockParameter
+    [Todo(Improve.Refactor, "Can make it struct?")]
+    public class BlockParameter : IEquatable<BlockParameter>
     {
-        private static readonly BlockParameter Earliest = new BlockParameter(BlockParameterType.Earliest);
+        public static BlockParameter Earliest = new BlockParameter(BlockParameterType.Earliest);
 
-        private static readonly BlockParameter Pending = new BlockParameter(BlockParameterType.Pending);
+        public static BlockParameter Pending = new BlockParameter(BlockParameterType.Pending);
 
-        private static readonly BlockParameter Latest = new BlockParameter(BlockParameterType.Latest);
+        public static BlockParameter Latest = new BlockParameter(BlockParameterType.Latest);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)